base_content_space.sol

```diff
@@ -78,6 +78,7 @@ contract BaseContentSpace is MetaObject, Container, UserSpace, NodeSpace, IKmsSp
     // override
     function setVisibility(uint8 _visibility_code) public onlyOwner {
         visibility = _visibility_code;
+        emit VisibilityChanged(address(this), 0x0, visibility);
```

Comment:
should we update the `version`?

Reply:
Yes, otherwise we might be looking for an event we do not see and draw the wrong conclusions